src/app/component/eco-news/components/news-list/news-list-gallery-view/news-list-gallery-view.component.scss

```diff
@@ -1,6 +1,6 @@
 $bp-smallest: 575px;
```

Comment:
I told you dont't touch these components until my pull request will be merged. You waste your time.

Reply:
I saw your pull request and econews gallery display correct. So I will delete this pull request.